app/controllers/date_history/reasons/earlier_controller.rb

```diff
@@ -0,0 +1,29 @@
+class DateHistory::Reasons::EarlierController < ApplicationController
+  include Projectable
+
+  def new
+  end
+
+  def create
```

Comment:
As mentioned - yes I think these two need authorization, same as the DateHistoryController